CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.10.8dev
 * [Fix] Fix edge case where `select` and other SQL keywords were not properly used to find where the user's query started, causing argument parsing issues (#973)
+* [Feature] Add option to completely disable named parameters in config (#972) and (#971)
```

Comment:
Also add the changed options (`0` for False etc)

Reply:
[Added changed options](https://github.com/ploomber/jupysql/pull/980/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR5)